### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by MariaMaximova**
Update issue templates (bug, feature)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_short_name.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug_short_name
```
Comment (OleksiiBulba) [inline .github/ISSUE_TEMPLATE/bug_short_name.md:2]: @MariaMaximova, what is the difference between `Bug report` and `Bug short name`?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: BUG
```
Comment (OleksiiBulba) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: I would suggest changing it to:
```suggestion
title: "[BUG] *Enter bug title here*"
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature]"
```
Comment (OleksiiBulba) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: I would suggest changing it to:
```suggestion
title: "[Feature] *Enter feature title here"
```

**Comment by MariaMaximova** [comment]
Added: Apply suggestions from code review

**Review by OleksiiBulba — APPROVED** [review]

**Pull request merged**